### ISSUE TITLE
refresh captcha not working

### ISSUE THREAD

**Issue by Priya-2016** [VahidN/DNTCaptcha.Core#130]
# Summary of the issue
We are using dtncapcha but refresh button not working. tried CDN link , included jquery.unobtrusive-ajax.min.js in project but still not working
<script type="text/javascript" src="http://ajax.aspnetcdn.com/ajax/mvc/3.0/jquery.unobtrusive-ajax.min.js"></script>

## Environment
asp.net
```
DNTCaptcha.Core version: 
.NET Core SDK version: 
IDE: 
```
visual studio 2019
## Example code/Steps to reproduce:
 <div class="form-group row">
                                    <label for="security" class="col-sm-4 col-form-label">Security Code</label>
                                    <div class="col-sm-8">
                                        <dnt-captcha asp-captcha-generator-max="999999"
                                                     asp-captcha-generator-min="111111"
                                                     asp-captcha-generator-language="English"
                                                     asp-captcha-generator-display-mode="ShowDigits"
                                                     asp-use-relative-urls="true"
                                                     asp-placeholder="Enter Security code"
                                                     asp-validation-error-message="Please enter the security code."
                                                     asp-font-name="Tahoma"
                                                     asp-font-size="20"
                                                     asp-fore-color="#333333"
                                                     asp-back-color="#ccc"
                                                     asp-text-box-class="text-box form-control col-sm-8"
                                                     asp-text-box-template="<span class='input-group-prepend'><span class='form-group-text'></span></span>{0}"
                                                     asp-validation-message-class="text-danger"
                                                     asp-refresh-button-class="fas fa-redo fa-lg"
                                                     asp-use-noise="false" />
                                    </div>
                                </div>
```
paste your core code
```
same as above
## Output:

```
![Capture](https://user-images.githubusercontent.com/90097344/153351815-f2ac35c0-4930-43bc-ac90-55af9f38e57f.JPG)

Exception message: nil
Full Stack trace:
```
No error but refresh feature not working

**Comment by VahidN** [comment]
- [There a full sample here](https://github.com/VahidN/DNTCaptcha.Core/tree/master/src/DNTCaptcha.TestWebApp)
- [You will need these dependencies](https://github.com/VahidN/DNTCaptcha.Core/blob/master/src/DNTCaptcha.TestWebApp/libman.json)
- [With this order](https://github.com/VahidN/DNTCaptcha.Core/blob/master/src/DNTCaptcha.TestWebApp/Views/Shared/_Layout.cshtml#L18)
